hiob-ludolf-centre-for-ethiopian-studies.csl

```diff
@@ -136,7 +136,7 @@
         <text variable="title" font-style="italic"/>
       </else-if>
       <else>
-        <text variable="title" strip-periods="false" quotes="false" prefix="‘" suffix="’,"/>
+        <text variable="title" strip-periods="false" quotes="true"/>
```

Comment:
yes done thank you

Quoting Sebastian Karcher <notifications@github.com>:

>> @@ -136,7 +136,7 @@
>>          <text variable="title" font-style="italic"/>
>>        </else-if>
>>        <else>
>> -        <text variable="title" strip-periods="false"  
>> quotes="false" prefix="?" suffix="?,"/>
>> +        <text variable="title" strip-periods="false" quotes="true"/>
>
> you'll probably want to keep the comma here. add `suffix=","`
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1981/files/5af95e353ecd0da3297fb4521c79c33ce99171e8#r57475747



-- 
Eugenia Sokolinski
Project Coordinator
Hiob Ludolf Centre for Ethiopian Studies
Asien Afrika Institut
Universit�t Hamburg
Alsterterrasse 1
20354 Hamburg

eugenia.sokolinski@uni-hamburg.de

http://www1.uni-hamburg.de/ethiostudies
http://www.traces.uni-hamburg.de